README.md

```diff
@@ -103,6 +103,224 @@ PSPDFKit.present('document.pdf', {
+
+  ```java
+package com.yourapp;
+
+import android.app.Application;
+import android.util.Log;
+
+import com.facebook.react.ReactApplication;
+import com.facebook.react.ReactInstanceManager;
+import com.facebook.react.ReactNativeHost;
+import com.facebook.react.ReactPackage;
+import com.facebook.react.shell.MainReactPackage;
+import com.react.pspdfkit.PSPDFKitPackage; // <--- import
+
+import java.util.Arrays;
+import java.util.List;
+
+public class MainApplication extends Application implements ReactApplication {
+
+  private final ReactNativeHost mReactNativeHost = new ReactNativeHost(this) {
+    @Override
+    protected boolean getUseDeveloperSupport() {
+      return BuildConfig.DEBUG;
+    }
+    
+    @Override
+    protected List<ReactPackage> getPackages() {
+       return Arrays.<ReactPackage>asList(
+           new MainReactPackage(), // <------- add comma
+           new PSPDFKitPackage()   // <------- add package
+        );
+    }
+  };
+  .....
+}
+  ```
+     
+12. Set default PSPDFKit theme. In `YourApp/android/app/src/main/res/values/style.xml` replace
+  ```xml    
+<style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
+  ```
+with
+  ```xml    
+<style name="AppTheme" parent="PSPDFCatalog.Theme.Light"> 
+  ```     
+13. Replace the default component from `index.android.js` with a simple touch area to present the bundled PDF:
+
+  ```javascript
+import React, { Component } from 'react';
+import {
+  AppRegistry,
+  StyleSheet,
+  NativeModules,
+  Text,
+  TouchableHighlight,
+  View
+} from 'react-native';
+
+var PSPDFKit = NativeModules.PSPDFKit;
+
+const DOCUMENT = "document.pdf";
+const LICENSE = "LICENSE_KEY_GOES_HERE";
+const CONFIGURATION = {
+  startPage : 3,
+  scrollContinuously : false,
+  showPageNumberOverlay : true,
+  grayScale : true,
+  showPageLabels : false,
+  pageScrollDirection : "vertical"
+};
+
+// Change 'YourApp' to your app's name.
+class YourApp extends Component {
+  _onPressButton() {
+    PSPDFKit.presentLocal(DOCUMENT, LICENSE, CONFIGURATION);
+  }
+
+  render() {
+    return (
+      <View style={styles.container}>
+      <Text style={styles.version}>{PSPDFKit.VERSION}</Text>
+        <TouchableHighlight onPress={this._onPressButton}>
+          <Text style={styles.text}>Tap to Open Document</Text>
+        </TouchableHighlight>
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: '#F5FCFF',
+  },
+  text: {
+    fontSize: 20,
+    textAlign: 'center',
+    margin: 10,
+  }
+});
+
+// Change both 'YourApp's to your app's name.
+AppRegistry.registerComponent('YourApp', () => YourApp);
+  
+  ```  
+
+Your app is now ready to launch. From `YourApp` directory run `react-native run-android`.
+
+#### Configuration
+
+To copy a pdf document to your local device storage:
```

Comment:
I'm not sure I'm following you, this configuration step is intended for users who want to load a file, a pdf document in this particular case, from PC to Android device connected via ADB. This has been mentioned because `react-native-pspdfkit` module does not come with any PDF document bundled.

`assets` may have a double interpretation in this case.
1. Access Android assets folder natively: this falls back to regular Android folder and React Catalog app provides an example.
2. Access Android assets folder via React: as of today, 8th of Nov, `react-native-fs` does not provide a way to access it, but there is an issue for it https://github.com/johanneslumpe/react-native-fs/issues/64